api/diff.go

```diff
@@ -52,7 +52,7 @@ func handleAPIDiffGet(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	store := shared.NewAppEngineDatastore(ctx)
+	store := shared.NewAppEngineCachedDatastore(ctx)
```

Comment:
I'd prefer `NewAppEngineDatastore` take a `cached` boolean param than 2 separate functions. Copy pasta is a little harder that way (all call sites have the param, rather than not knowing about an overload method).

Reply:
Good point! will do.